common/src/main/java/fi/hel/verkkokauppa/common/rest/RestServiceClient.java

```diff
@@ -29,13 +29,21 @@
     public JSONObject makeGetCall(String url) {
         WebClient client = getClient();
         JSONObject response = queryJsonService(client, url);
-        return response;
+        if (response == null) {
```

Comment:
These fixed NullPointerException when JsonObject is tried to make from null value.